core/storage/statistics/gae_models.py

```diff
@@ -622,6 +622,9 @@ def create(
             params: dict. Current parameter values, map of parameter name
                 to value.
             play_type: str. Type of play-through.
+
+        Returns:
+            str. New unique ID for this entity class.
```

Comment:
Do you mean for the entity instance? The ID is unique to just the instance, right (not the entire class)?

Reply:
Sorry, it was a mistake on my part. I have now refactored the code. PTAL!